paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/SavedPaymentMethodRowButtonScreenshotTest.kt

```diff
@@ -0,0 +1,176 @@
+package com.stripe.android.paymentsheet.ui
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.stripe.android.model.CardBrand
+import com.stripe.android.model.PaymentMethod
+import com.stripe.android.model.PaymentMethodFixtures
+import com.stripe.android.paymentsheet.DisplayableSavedPaymentMethod
+import com.stripe.android.screenshottesting.FontSize
+import com.stripe.android.screenshottesting.PaparazziRule
+import com.stripe.android.screenshottesting.SystemAppearance
+import com.stripe.android.utils.screenshots.PaymentSheetAppearance
+import org.junit.Rule
+import kotlin.test.Test
+
+internal class SavedPaymentMethodRowButtonScreenshotTest {
```

Comment:
I'm still not fully convinced by these tests. I think we need one test that tests the layout, but it feels like the other part you're trying to test is the icons, which we should probably just unit test.

I also thought that SPMs didn't support a subtitle. So we can probably remove that functionality, right?

Reply:
gotcha. I can update the tests. 

Not sure what you're referring to re: subtitles. I agree that there are no subtitles for SPMs but there already isn't functionality for that

Reply:
There are lots of screenshot tests that have subtitle.  I left a comment on klarna, but I don't think any of these could have subtitle.

Reply:
Updated the screenshot tests to just test a saved visa in selected, unselected + disabled states.

I don't think that unit tests for the icons would be very useful, because they would just be testing that a when statement is returning a specific resource. 

The screenshot tests with subtitles are from PaymentMethodRowButton screenshot tests, as a result of refactoring some styling from NewPaymentMethodRowButton into PaymentMethodRowButton. None of the saved PMs row buttons have subtitles